CONTRIBUTING.md

```diff
@@ -209,6 +209,52 @@ this can be suppressed by adding this near the top of your file:
 #pragma warning disable SA1401 // Fields must be private
 ```
 
+### SafeHandles
+
+Safe handles should follow a few rules :
+* They should have an empty constructor that does nothing. (The marshaller will use this one when it need to create a SafeHandle)
+* They should have a constructor allowing to reuse pre-existing handles.
+* They should have a static field for each invalid values for easy access.
+
+Example :
```

Comment:
Good idea. I'd place it in a `/docs/` folder following https://gist.github.com/davidfowl/ed7564297c61fe9ab814 what do you think ?